.github/workflows/build-nightly-release-workflow.yml

```
@@ -163,6 +163,10 @@ jobs:
     with:
       ice_version: ${{ needs.set-version.outputs.semver_version }}
       source_url: "${{ inputs.repository_url }}/nuget-nightly/"
+      repository_url: "${{ inputs.repository_url }}/maven-nightly/"
```

```suggestion
      repository_url: "${{ inputs.repository_url }}/nightly/"
```

I assume we want to use the raw repository.